content/docs/secure/encryption/deployment.md

```diff
@@ -0,0 +1,168 @@
+
+## Secrets and Config Maps
+
+Apart from any secrets and config maps described in the CSI driver documentation, these resources should be created for Encryption:
+
+### Secret *encryption-license*
+
+Request a trial license following instructions on the [License page](../../../license). You will be provided with a YAML file similar to:
+
+```yaml
+apiVersion: v1
+data:
+  license: k1FXzMDZodGNnK4I12Alo4UvuhLd+ithRhuLz2eoIxlcMSfW0xJYWnBiNMvTUl8VdGmR5fsvs2L6KqPfpIJk4wOzCxQ9wfDIJuYqrwV0wi2F2lzb1Hkk7O7/4r8cblPdCRJWfbg8QFc2BVtl4PZ/pFkHZoZVCbhGDD1MsbI1CiKqva9r9TBfswSFnqv7p3QXgbqQov8/q/j2+sHcvFF3j4kx+q1PzXoRNxwuTQaP4VAvipsQNAU5yV2dos2hs4Y/Ltbtreu/vrRGUaxvPbass1vUtIOJnvKkfbp53j8PFJGGISMYvYylUiD7TpoamxT/1I6mkjgRds+tEciMvutqDpmKEtdyp3vBjt4Sgd07ptvsdBJlyRAYb8ZPX9vXr4Ws
+kind: Secret
+metadata:
+  name: edit_name
+  namespace: edit_namespace
+```
+
+Set `name` to `"encryption-license"` and `namespace` to your driver namespace and apply the file:
+
+```shell
+kubectl apply -f <license yaml file name>
+```
+
+### Secret *vault-auth*
+
+A secret with the AppRole credentials used by Encryption to authenticate to the Vault server.
+
+> Set `role_id` and `secret_id` to the values provided by the Vault server administrator.
+
+> If a self-managed test Vault instance is used, generate role ID and secret ID following [these steps](../vault/#set-role-id-and-secret-id-to-the-role).
+
+```shell
+cat >auth.json <<EOF
+{
+	"role_id": "<role ID>",
+	"secret_id": "<secret ID>"
+}
+EOF
+
+kubectl create secret generic vault-auth -n <driver namespace> --from-file=auth.json -o yaml --dry-run=client | kubectl apply -f -
+
+rm -f auth.json
+```
+In this release, Encryption does not pick up modifications to this secret in runtime, unless it needs to re-login which happens at:
+- CSI Driver startup
+- an authentication error from the Vault server
+- client token expiration
+
+In all other cases, to apply new values in the secret (e.g., to use another role), the CSI driver must be restarted.
+
+### Secret *vault-cert*
+
+A secret with TLS certificates used by Encryption to communicate with the Vault server.
+
+> Files *server-ca.crt*, *client.crt* and *client.key* should be in PEM format.
+
+```shell
+kubectl create secret generic vault-cert -n <driver namespace> \
+	--from-file=server-ca.crt --from-file=client.crt --from-file=client.key \
+	-o yaml --dry-run=client | kubectl apply -f -
+```
+In this release, Encryption does not pick up modifications to this secret in runtime. 
+To apply new values in the secret (e.g., to update the client certificate), the CSI driver must be restarted.
+
+### ConfigMap *vault-client-conf*
+
+> Replace fields in *client.json* with your values. See the table below for available fields.

```

Comment:
Replace "the table below" with "this table"

Reply:
The sentence containing "the table below" refers to the table that does not immediately follow the sentence. Would it be appropriate to use "this" is that context?